src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/ModuleAntiHunger.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015 - 2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.ccbluex.liquidbounce.features.module.modules.exploit
+
+import net.ccbluex.liquidbounce.event.Listenable
+import net.ccbluex.liquidbounce.event.events.PacketEvent
+import net.ccbluex.liquidbounce.event.events.PlayerTickEvent
+import net.ccbluex.liquidbounce.event.handler
+import net.ccbluex.liquidbounce.features.module.Module
+import net.ccbluex.liquidbounce.features.module.Category
+import net.minecraft.network.packet.c2s.play.ClientCommandC2SPacket
+import net.minecraft.network.packet.c2s.play.PlayerMoveC2SPacket
+
+/**
+ * AntiHunger Module
+ *
+ * Prevents hunger from decreasing, will flag anticheats.
+ * Tested on PurityVanilla. Ran about 2000 blocks in the nether and hunger did not decrease.
+ */
+
+object ModuleAntiHunger : Module("AntiHunger", Category.EXPLOIT) {
+    private var ignorePacket = false
```

Comment:
What purpose does this variable has? It is being set, but never used.

Reply:
that can probably be removed